packages/service-worker-mock/models/Headers.js

```diff
@@ -1,6 +1,56 @@
 // stubs https://developer.mozilla.org/en-US/docs/Web/API/Headers
 
-class Headers extends Map {}
+class Headers {
+  constructor(meta) {
+    if (meta && meta instanceof Headers) {
+      this._map = new Map(meta._map);
+    } else if (meta && typeof meta === 'object') {
+      const iterable = Object.keys(meta).map(key => ([key, meta[key]]));
```

Comment:
You could use `Object.entires()` here unless you have support concerns (we're transpiling, so it works for us).